pkg/onboarding/usecases/shortcode.go

```diff
@@ -0,0 +1,155 @@
+package usecases
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/savannahghi/onboarding/pkg/onboarding/application/dto"
+	"github.com/savannahghi/onboarding/pkg/onboarding/application/extension"
+	"github.com/savannahghi/onboarding/pkg/onboarding/application/utils"
+	"github.com/savannahghi/onboarding/pkg/onboarding/infrastructure/services/crm"
+	"github.com/savannahghi/onboarding/pkg/onboarding/infrastructure/services/engagement"
+	pubsubmessaging "github.com/savannahghi/onboarding/pkg/onboarding/infrastructure/services/pubsub"
+	"github.com/savannahghi/onboarding/pkg/onboarding/repository"
+	log "github.com/sirupsen/logrus"
+	"gitlab.slade360emr.com/go/commontools/crm/pkg/domain"
+)
+
+const (
+	// The right copy of replies will be availed later
+	bewellUserReplyMessage = "We have received your request and one of our representatives will reach out to you. Thank you"
+	newSenderReplyMessage  = "We have received your request and one of our representatives will reach out to you. Thanks you.\n\nDid you know you can now view your medical cover and benefits on Be.Well. To get started, Download Now https://bwl.mobi/1bvf"
+)
```

Comment:
Is it possible to talk with @vincentjosephmichuki and the design team to give you good copies?

Reply:
He already knows about that. He said theylll be provided a soon as they are ready